partner_company_type/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
__init__.py files still by convention have the License line, only the copyright lines were dropped